### PR TITLE
WEB: Update FAQ

### PR DESCRIPTION
**Pull request by criezy**
Some of the answers in our FAQ on the web site are outdated (for example on making new games or on AGS being closed source). Here are suggestions to improve the FAQ. Feel free to disagree with some of my changes or propose better formulations.

### CODE REVIEW

**Comment by rootfather** [comment]
The changes you made to the FAQ are very good if you ask me.

I have a question regarding the supported systems: While working on the German translation of the FAQ (slowly but steady :)) I noticed that Windows 95 is listed as supported system. Is this still true or an outdated information? I don't have a Windows 95 CD anymore to test this on a virtual machine.

**Comment by lordhoto** [comment]
Win95 should still work.

**Comment by lordhoto** [comment]
The changes look good to me. I added a nitpicking comment to one place though :-).

**Comment by rootfather** [comment]
Thanks for the information. I should have a Windows 95 or 98 disk laying
around somewhere around here. I'll give it a try just to check it.

But anyway, we should add Windows versions after XP until Windows 10 to the
FAQ.

**Comment by criezy** [comment]
I made suggested by lordhoto to use 'Macintosh' rather than 'mac' (it actually appeared in a few other places in the FAQ and I changed it everywhere).

In the same commit I also changed 'Mac OS X' into 'OS X' since this is the official name now. That one might be a bit controversial though since this is in the list of platforms we support, and we support old versions that are officially named 'Mac OS X'. But I didn't think it was worse listing both.

**Comment by criezy** [comment]
I now updated the list of supported Windows version to list the recent ones. I have to wonder though if we should really list all the versions. We don't do that for other OS's. Or we could just write 'all Windows versions from Windows 95 up to Windows 10'.

**Comment by criezy** [comment]
If there is no objections I will squash the three commits and push to origin/master on Sunday evening.

File: data/faq-xml.xml [hunk 1]
```diff
@@ -527,7 +553,9 @@
 		These are not full games but reencoded cutscene (video) packs. To run the games you
 		still need the original disks. Originally cutscenes were encoded in a proprietary
 		format but Revolution Software kindly permitted us to reencode them into
-		DXA and distribute them freely on our site.
+		DXA and distribute them freely on our site. Note however that recent ScummVM versions
```
Comment (RichieSams) [inline data/faq-xml.xml]: One small suggestion for grammer:

"Note, however, that recent ScummVM versions support using the original cutscene files directly. Therefore, you don't have to use the ones from our site anymore."

**Comment by criezy** [comment]
I have now pushed the commit to master.

**Pull request closed without merging**